LibraBFT/ImplShared/Consensus/Types.agda

```diff
@@ -31,11 +31,13 @@ open import Data.String using (String)
 -- is no epoch change.
 
 module LibraBFT.ImplShared.Consensus.Types where
-  open import LibraBFT.ImplShared.Base.Types                 public
-  open import LibraBFT.Abstract.Types.EpochConfig UID NodeId public
-  open import LibraBFT.ImplFake.NetworkMsg                   public
-  open import LibraBFT.ImplShared.Consensus.Types.EpochIndep public
-  open import LibraBFT.ImplShared.Consensus.Types.EpochDep   public
+  open import LibraBFT.ImplFake.NetworkMsg                       public
+  open import LibraBFT.ImplShared.Base.Types                     public
+  open import LibraBFT.ImplShared.Consensus.Types.EpochIndep     public
+  open import LibraBFT.ImplShared.Consensus.Types.MetaEpochIndep public
```

Comment:
This file doesn't exist, and doesn't appear to be needed.

Reply:
It is the new home for the definition of `Output`. I moved `MetaEpochIndep` to the `ImplShared` directory, but forgot to track it -- this has been fixed now.